meta-lmp-base/recipes-containers/libnvidia-container/libnvidia-container_1.10.0.bbappend

```diff
@@ -0,0 +1 @@
+RDEPENDS:${PN}:append:tegra = " tegra-libraries-cuda"
```

Comment:
Have it under dynamic layers, otherwise bitbake will complain when meta-tegra is not there.

Might also not need the 'tegra' override here since it is only relevant for tegra anyway.

Reply:
right! fixed